packages/components/tests/vue/components/word-list/word-language-panel.test.js

```diff
@@ -594,7 +608,7 @@ describe('word-language-panel.test.js', () => {
     expect(result[2].targetWord).toEqual('cupidinibus')
   })
 
-  it('26 WordLanguagePanel - method downloadList executes Download.collectionToCSV, Download.downloadBlob and hides downloadAll confirmation box', () => {
+  it.skip('26 WordLanguagePanel - method downloadList executes Download.collectionToCSV, Download.downloadBlob and hides downloadAll confirmation box', () => {
```

Comment:
why are we skipping this test now?

Reply:
I need to update it because this features was moved to a separate component - fixed